manifests/j/JCTOT/ONOIPABEGGARS/1.2.1/JCTOT.ONOIPABEGGARS.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: JCTOT.ONOIPABEGGARS
+PackageVersion: 1.2.1
+Installers:
+- InstallerUrl: https://cdn-files.gamejolt.net/HHn6y5FgqVQEb3tiTOoEMA==,1685051084/data/games/12/112/768112/files/63ae587f98cbb/baldi-surroundedipabeggars-windows-copy-copy-copy.zip

```

Comment:
This URL appears to be a temporary URL which is unique each time you visit the site. The `1685051084` indicates the time it will expire - which is roughly 19 hours from the time of this review. When I visited the URL I got a different number which was 24 hours from now.

Reply:
This host's configuration seems designed to prevent downloading without visiting their page and viewing its ads. Does the developer provide any official mirrors?

Reply:
No.

Reply:
If there are no official mirrors, we will be unable to add this package due to the temporary URL.
<!--
Close with reason: No static or vanity URLs to the package;
-->